cheatsheets/Python_Debugger.rb

```diff
@@ -9,7 +9,12 @@
 
         entry do
             name 'Start pdb inside a python script'
-            command 'import pdb;pdb.set_trace() '
+            command 'import pdb;pdb.set_trace()'
+        end
+
+        entry do
+            name 'Same as `import pdb;pdb.set_trace()` (New in Python 3.7)'
+            command 'breakpoint()'
```

Comment:
You can add this as a second command in the other entry. No need to create a new entry for it.

Reply:
I'll make this change.

Reply:
Yes, this is also good.